use_cases/eluc/predictors/sklearn/sklearn_predictor.py

```diff
@@ -24,10 +24,9 @@ def __init__(self, model_config: dict):
         Model config contains the following:
         features: list of features to use for prediction (optional, defaults to all features)
         label: name of the label to predict (optional, defaults to passed label during fit)
+        Any other parameters are passed to the model.
         """
-        self.features = model_config.get("features", None)
-        self.label = model_config.get("label", None)
-
+        self.config = model_config
```

Comment:
Some config refactoring so we can save our training arguments for reproducibility